.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
- [ ] You did a great work adding the `eslintrc.json` file, but you should also update your `linters.yml` in the `.github/workflows` directory so GitHub can check it also like Html and CSS.

Please update your `linters.yml` file using this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file. 

As you see in the pic there is no check for `eslintrc`
![image](https://user-images.githubusercontent.com/35535130/157055002-15711345-7c3e-4310-ae41-f337482adaa2.png)